src/index.html

```diff
@@ -16,7 +29,87 @@
       href="./style.css"
     />
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body class="body">
+    <header class="header">
+      <a
+        href="#"
+        class="logo_link"
+      >
+        <img
+          class="logo"
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav">
```

Comment:
it isn't good practice name class like tag, class name should have logical meaning, what is it?

example:

```
        <ul class="nav_list">
          <li class="nav_item">
            <a
              class="nav_link is-active"
              href="#"
            >
              Apple
            </a>
          </li>
```